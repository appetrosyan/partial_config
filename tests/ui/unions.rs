#[derive(partial_config::HasPartial)]
#[repr(C)]
pub union Conf {
    f1: u32,
    f2: f64,
}

fn main() {}
