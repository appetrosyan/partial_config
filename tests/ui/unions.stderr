error: Data unions are not supported.

         = help: Data unions are not usually used in Safe Rust, even though they could be, this is discouraged in favour of Enums, which are not supported either. Consider using a `struct` instead.

 --> tests/ui/unions.rs:3:11
  |
3 | pub union Conf {
  |           ^^^^
